src/test/java/br/com/caelum/vraptor/test/VRaptorNavigationSimpleScenariosTest.java

```diff
@@ -155,4 +159,18 @@ public void shouldPassQueryString() {
 		assertEquals(value, result.getObject(name));
 		assertEquals(query, result.getObject("query"));
 	}	
+	
+	@Test
+	public void shouldJsonConsumes() {
+		String value1 = "123";
+		String value2 = "abc";
+		String json = String.format("{\"value2\":\"%s\",\"value1\":\"%s\"}", value2, value1);
+		VRaptorTestResult result = navigate()
+									.post("/test/testJson")
+									.addHeader("Content-Type", "application/json; charset=utf-8")
+									.setContent(json.getBytes())
```

Comment:
i'd throw exception if user calls setContent multiple times, what do you think?
as i can see, currently we just overwrite it